src/ida_lunde_naalsund_ex/ex05/myrand.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Ida Lunde Naalsund"
+__email__ = "idna@nmbu.no"
+
+
+class LCGRand:
+    """Implementation of a linear congruential generator
+    that returns random numbers.
+    """
+
+    def __init__(self, seed):
+        """Initialises the class with given constants a and m.
+
+        Parameters
+        ----------
+        seed : int
+            The seed that the generating is based on.
+        """
+        self._hidden_state = seed
+        self.slope = 7 ** 5
+        self.congruence_class = 2 ** 31 - 1
+
+    def rand(self):
+        """Generates a single random number based on a seed given by the user.
+
+        Returns
+        -------
+        random_number : int
+            The generated number.
+        """
+        self._hidden_state *= self.slope
+        self._hidden_state %= self.congruence_class
+
+        return self._hidden_state
+
+    def random_sequence(self, length):
+
```

Comment:
It would be good practice for understanding what `RandIter` does to write a docsting here